org.codeberg.dnkl.foot.yml

```diff
@@ -0,0 +1,53 @@
+app-id: org.codeberg.dnkl.foot
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+command: foot.sh
+
+finish-args:
+  - --socket=wayland
+  - --filesystem=xdg-config/foot:ro
```

Comment:
What exactly is this one for?

Reply:
foot reads it's configuration from `~/.config/foot/foot.ini`

Reply:
Flatpak has it's own config directory, so this is not needed

Reply:
> foot reads it's configuration from `~/.config/foot/foot.ini`

Did you actually check it was needed? this is the standar xdg path outside flatpak

One way to check is not have that permission and see if the settings are retained across launches.